phpunit.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    bootstrap="vendor/autoload.php"
```

Comment:
We could probably switch it to symfony's runtime, but that's a topic for a later date.